cmd/template/cluster/provider/templates/azure/azure_machine_template.yaml.tmpl

```diff
@@ -21,7 +21,7 @@ spec:
       - diskSizeGB: 256
         lun: 0
         nameSuffix: etcddisk
-      identity: SystemAssigned
+      identity: None
```

Comment:
?

Reply:
my reasoning is that using SystemAssigned identity will fail if a new machine is added for whatever reason. The new machine will never work. My upstream PR got merged but it's still not released. So I thought that using None would help here. Using None means using the env vars that are passed to the capz controllers, I figured that was good enough for our alpha release